src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.targets

```diff
@@ -27,6 +27,10 @@
     <ProjectCapability Include="Mobile" />
     <ProjectCapability Include="Android" />
     <ProjectCapability Include="AndroidApplication" Condition="$(AndroidApplication)" />
+    
+    // See https://work.azdo.io/1112733
```

Comment:
Not sure how MSBuild deals with "content" between items in an ItemGroup, but it's likely not a good idea, regardless.  This "comment" should be moved into the *actual* XML comment on the following line.

Reply:
doh!